Translator/frmTranslate.cs

```diff
@@ -239,28 +239,14 @@ private void dgvSection_CellValueChanged(object sender, DataGridViewCellEventArg
                 string strPage = item.Cells[cboFile.Text == "books.xml" ? "Code" : "Page"].Value.ToString();
                 bool blnHasNameOnPage = cboFile.Text == "qualities.xml";
                 string strNameOnPage = blnHasNameOnPage ? item.Cells["NameOnPage"].Value.ToString() : string.Empty;
-                XmlNode xmlNodeLocal = _objDataDoc.SelectSingleNode(strBaseXPath + "/id[text()=\"" + strId + "\"]/..") ??
-                                       _objDataDoc.SelectSingleNode(strBaseXPath + "/name[text()=\"" + strEnglish + "\"]/..");
+                XmlNode xmlNodeLocal = _objDataDoc.SelectSingleNode(strBaseXPath + "/*/id[text()=\"" + strId + "\"]/..") ??
+                                       _objDataDoc.SelectSingleNode(strBaseXPath + "/*/name[text()=\"" + strEnglish + "\"]/..");
                 if (xmlNodeLocal == null)
                 {
                     xmlNodeLocal = _objDataDoc.SelectSingleNode(strBaseXPath + "/*[text()=\"" + strEnglish + "\"]");
                     if (xmlNodeLocal?.Attributes != null)
                     {
                         xmlNodeLocal.Attributes["translate"].InnerText = strTranslated;
-                        XmlAttribute objAttrib = xmlNodeLocal.Attributes?["translated"];
-                        if (objAttrib != null)
-                        {
-                            if (!flag)
-                                xmlNodeLocal.Attributes.Remove(objAttrib);
-                            else
-                                objAttrib.InnerText = bool.TrueString;
-                        }
-                        else if (flag)
-                        {
-                            objAttrib = _objDataDoc.CreateAttribute("translated");
-                            objAttrib.InnerText = bool.TrueString;
-                            xmlNodeLocal.Attributes.Append(objAttrib);
-                        }
```

Comment:
This needs to be rethought. The `translated` tag is there for if something's English name matches its translated name, which is the case sometimes in German and French. Since stuff without a translation gets highlighted in a different color, this makes sure those highlights don't happen when they aren't supposed to.

Reply:
This part sets the items to be displayed in'translated'in the list display when'translate tag' is modified.
However,'translated' does not have in'ja-jp_data.xml', it is an item only for display.
If you reselect the section and redisplay it, it will be set correctly.
If you leave this part, the attribute "translated" will be registered in'ja-jp_data.xml'.
These should be unnecessary data in ja-jp_data.xml.
To fix it in its original form, it was necessary to save the changes in xml, redisplay the view, and select the column that was edited immediately before, but I could not incorporate it well.

Reply:
Having `translated` attributes appear in `ja-jp_data.xml` should not be a problem though, right? And if the code is working the way it should, all it would be doing is creating and/or removing that attribute from items that are (or are not) translated, it should not affect the translations themselves.

Reply:
If'translated = true'is included in xml,'Translated' will not be changed as'True' unless you enter'False' in'Translated' after making the contents of'English' and'Text' the same. 
It sounds strange in behavior, but I think it's unlikely that you'll revert to'False', and if it doesn't affect the conversion in'Chummer', that's okay.

In my opinion, it's best not to include unnecessary data in xml
I hope someone will remove this unwanted attribute in the future.